Source/Core/CameraLimiter.js

```diff
@@ -0,0 +1,70 @@
+define([
+    '../Core/Cartesian3',
+    '../Core/Check',
+    '../Core/defaultValue',
+    '../Core/defined',
+    '../Core/HeadingPitchRoll',
+    '../Core/Math'
+], function(
+    Cartesian3,
+    Check,
+    defaultValue,
+    defined,
+    HeadingPitchRoll,
+    CesiumMath) {
+    'use strict';
+
+    /**
+     * The CameraLimiter is used to limit a {@link Camera}'s position (using a bounding object).
+     *
+     * If set, the camera of this limiter will be constrained by location.
+     * The camera ignores position limits in {@link SceneMode#SCENE2D} view and {@link SceneMode#COLUMBUS_VIEW}.
```

Comment:
"Object used to define bounds limiting the camera's position when in 3D mode"?

Link to `Camera.cameraLimiter`. 

Remove:
> If set, the camera of this limiter will be constrained by location.
> The camera ignores position limits in {@link SceneMode#SCENE2D} view and {@link SceneMode#COLUMBUS_VIEW}.

That should be explained in `Camera.cameraLimiter`